ts-client/src/dlmm/index.ts

```diff
@@ -184,15 +185,31 @@ export class DLMM {
     const program = new Program(IDL, LBCLMM_PROGRAM_IDS[cluster], provider);
 
     try {
-      const [lbPairKey] = deriveLbPair2(
+      const [lbPair2Key] = deriveLbPair2(
         tokenX,
         tokenY,
         binStep,
         baseFactor,
         program.programId
       );
-      await program.account.lbPair.fetch(lbPairKey);
-      return true;
+      const account2 = await program.account.lbPair.fetchNullable(lbPair2Key);
+      if (account2) return lbPair2Key;
```

Comment:
Can we return true? When account doesn't exist it return false, let's make it consistent

Reply:
Can we do `PublicKey | null`? I need to return the pool address for the warning of exists pool redirect link